radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -282,3 +284,27 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 
 	return 0;
 }
+
+/**
+ * Resets microcontroller of the antenna
+ *
+ *  @return 0 for success, non-zero for failure. See hal/errors.h for details.
+ */
```

Comment:
Could consider adding more detail within this doc block e.g. is it a hard or soft reset? Does it reset one micro or both? Could also place the comment about the relevant user manual section.

For example (idk if this stuff is true but just an example):
```c
 * Resets both internal microcontrollers on the antenna.
 *
 * Performs a hard reset of both microcontrollers that control the antenna subsystem. Transmits
 * commands over I2C to each MCU individually. Antenna will require **X** ms to fully reboot and be
 * active again. See section 6.2 of the Antenna System User Manual for more information.
 *
 * @return 0 for success, non-zero for failure. See hal/errors.h for details.
```